src/plot/series/abstract-series.js

```diff
@@ -39,7 +39,10 @@ const propTypes = {
   ...getScalePropTypesByAttribute('color'),
   width: PropTypes.number,
   height: PropTypes.number,
-  data: PropTypes.arrayOf(PropTypes.object),
+  data: PropTypes.arrayOf(PropTypes.oneOfType([
```

Comment:
add some documentation/testing to show that this case is supported?

Reply:
you mean testing that the warning is not triggered anymore? I did test this manually, but I am not sure how to achieve this through automated tests.

Reply:
more like add a test case demonstrating that this is a valid way to input data (I don't remember writing any but I might have?) and add a note in the documentation showing that this is a supported case (again I may have already done that but i don't think i did??). If it's the case that I am being dumb and these things already exist than this gets a stamp